build_runner/pubspec.yaml

```diff
@@ -40,3 +40,5 @@ dev_dependencies:
   test: ^0.12.0
   test_descriptor: ^1.0.0
   test_process: ^1.0.0
+  test_common:
+    path: ../test_common
```

Comment:
Can we make this `_test_common` to make it clear that we don't intend for it to be published?

Reply:
Done.